stable/rbac-manager/values.yaml

```diff
@@ -3,6 +3,7 @@ image:
   tag: v0.9.4
   pullPolicy: Always
   imagePullSecrets: []
+#   - myRegistryKeySecretName
```

Comment:
This seems like an unnecessary comment here.

Reply:
It helps so that you don't have to read the manifest. Some helm charts include the whole yaml which means you'd have to add `- name: myRegistryKeySecretName`

I don't have a strong opinion on this, I can remove it if you want